dev-docs/howto/vpn/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: vpn
+description: A VPN server for Constellation
+
+type: application
+
+version: 0.1.0
+
+appVersion: "v2.13.0"  # target Constellation version
```

Comment:
Since this is likely very loosely coupled and we don't want to confuse people about the compatibility I think that we should drop the optional `appVersion` field.